tasks/install-on-Darwin.yml

```diff
@@ -7,11 +7,18 @@
     name: buildkite/buildkite
     state: present
 
+# Workaround for currently broken homebrew release of 2.3.0
+- name: Checkout the latest working commit of homebrew
+  command: git checkout 2.2.17 
+  args:
+    chdir: /usr/local/Homebrew
+
 - name: Install buildkite-agent
   homebrew:
     name: buildkite-agent
-    state: latest
-    update_homebrew: yes
+    update_homebrew: no
+  environment:
```

Comment:
Please do one or the other, not both? It's confusing, and we won't know which one works.